mmv1/products/documentaiwarehouse/Location.yaml

```diff
@@ -0,0 +1,101 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: 'Location'
+base_url: 'projects/{{project}}/locations/us:initialize'
```

Comment:
`us` as a location is hardcoded in the url. Is `us` the only allowed location? Should it be replaced with `{{location}}`?

Reply:
Thanks for pointing it out, I'll replace it with {{location}} now